mobile/lib/services/preview_video_store.dart

```diff
@@ -34,12 +35,18 @@ import "package:photos/utils/exif_util.dart";
 import "package:photos/utils/file_key.dart";
 import "package:photos/utils/file_util.dart";
 import "package:photos/utils/gzip.dart";
+import "package:photos/utils/network_util.dart";
 import "package:photos/utils/toast_util.dart";
 import "package:shared_preferences/shared_preferences.dart";
 
+const _maxRetryCount = 3;
+
 class PreviewVideoStore {
   final LinkedHashMap<int, PreviewItem> _items = LinkedHashMap();
   LinkedHashMap<int, PreviewItem> get previews => _items;
+  late Set<int> _failureFiles;
```

Comment:
It might be better to either define this as emptySet or nullable.
Ideally, late variables should be initialized in the construction phase. Otherwise there's a risk of running into late variable not initialized error